.devcontainer/devcontainer.json

```
@@ -1,28 +1,31 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options,
 // see documentation located at the following URL:
 // https://github.com/OpenINF/docker-fisher/tree/main/library-scripts/docs
-// For format details, see https://aka.ms/devcontainer.json. For config options, see the
-// README at: https://github.com/devcontainers/templates/tree/main/src/docker-outside-of-docker
+// For config options, see the README at:
+// https://github.com/devcontainers/templates/tree/main/src/docker-outside-of-docker
 {
-	"name": "Docker outside of Docker",
-	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-    "image": "mcr.microsoft.com/devcontainers/javascript-node:18-bullseye"
+  "name": "Docker outside of Docker",
```

```suggestion
  "name": "Docker outside of Docker",
```

my confidence that this is the correct name is not high&hellip;